lib/carrierwave_data_uri/mount.rb

```diff
@@ -8,9 +8,13 @@ def mount_uploader(column, uploader=nil, options={}, &block)
         super
 
         class_eval <<-RUBY, __FILE__, __LINE__+1
+          attr_accessor :#{column}_data_uri
+          attr_accessor :#{column}_data_uri_cache
           attr_accessor :#{column}_data_filename, :#{column}_data_mimetype
 
           def #{column}_data_uri=(data)
+            return if data.blank?
+            self.#{column}_data_uri_cache = data
```

Comment:
In general, I think we can store data to some variable, but I don't like `_data_uri_cache` name.
I think it will be better to store it to `@#{column}_data_uri` and add `attr_reader :#{column}_data_uri`